automation/_input/04_Powers_SAMPLE.yaml

```diff
@@ -1,31 +1,41 @@
 Template: # Power name        # Must be unique
-    Type: One of              # Passive, Vulny, Major, Minor, Adversary, Free, House. 
-    # Note: Type is required. For Companion actions, type indicates type req for dealer
+    Type: One of              # Required. Passive, Vulny, Major, Minor, Adversary, Free, House.
     Category: One or list     # One string or list in sub-cat order
+    Mechanic: Descrip         # Rule/effect. Save info added after. No ending periods.
+    Description: Descrip      # Flavor: 1-2 sentences
     XP: X
     PP: X                     # For multiple, implies 'OR'. Use [A, B] or `-`
     Prereq:                   # For multiple, implies 'OR'. Use [A, B] or `-`
         Role: X
-        Level: X              # Integer               
-        Skill: X > 1          #              
+        Level: X              # Integer
+        Skill: X > 1          #
         Power:                # Must match unique power name
-    To Hit: X                 # If not specified, assumed character's primary skill
-    Damage: X                 # Integer           
+    StatOverride:             # One of AGL, STR, Finesse, etc. followed by number
+        Stat: X
+    ToHit: X                  # If not specified, assumed character's primary skill
+    Damage: X                 # Integer
     Range: X                  # Integer of spaces. 0 for self, 1 for melee
     AOE: Shape X              # Aura, cone, or line followed by integer for size
     Target: X                 # Integer, # of targets
-    Mechanic: Descrip         # Rule/effect. Save info added after. No ending periods.
+    Options: X                # Optional Descrip of options. Replaced by selection.
     Save:                     # Separated here, merged in md
         Trigger: One of       # Once, On hit, On starting turn, etc. Sentence start.
         DR: X                 # If primary skill modifier of actor, none [3-(MOD/2)]
         Type: MOD             # AGL, CON, etc
         Fail: Descrip         # effect on fail
         Succeed: Descrip      # effect on success. Assume no access to above
-    Description: Descrip      # Flavor: 1-2 sentences             
     Tags: One or list
 
+Dumb:
+    Type: Vulny
+    Category: Roleplay
+    StatOverride:
+        INT: -2
+    Description: You're dumb as rocks. We're sure you make up for it in other areas.
+    XP: -2
```

Comment:
Do we want to create a -1 minor and -2 major for each Attribute? I can work on that using this as a template.

Reply:
An alternate implementation I'd have to think about is using the Options and Choice parameters, but that would take a little more work on my part. Meaning one Attrib vulny, but choosing the attrib on selection. The easier thing would be to copy/paste the above 6 times and make edits accordingly